tools/nimgrep.nim

```diff
@@ -50,44 +58,132 @@ Options:
                -b:N   print N lines of leading context before every match
   --context:N, -c:N   print N lines of leading context before every match and
                       N lines of trailing context after it
+  --sortTime          order files by the last modification time -
+       -s[:asc|desc]  - ascending (default: recent files go last) or descending
   --group, -g         group matches by file
   --newLine, -l       display every matching line starting from a new line
+  --limit[:N], -m[:N] limit max width of lines from files by N characters (80)
+  --fit               calculate --limit from terminal width for every line
```

Comment:
remove in favor of --cols:auto (see https://github.com/nim-lang/Nim/pull/15612/files#r509810340)

Reply:
makes sense, thank you.